lib/checks/aria/required-children.js

```diff
@@ -51,7 +51,7 @@ function missingRequiredChildren(node, childRoles, all, role) {
 
 		// remove 'textbox' from missing roles if combobox is a native input
 		var textboxIndex = missing.indexOf('textbox');
-		if (textboxIndex >= 0 && node.tagName === 'INPUT' && node.type === 'text') {
+		if (textboxIndex >= 0 && node.tagName === 'INPUT' && ['text', 'search'].indexOf(node.type) >= 0) {
```

Comment:
I think there are probably other types that make sense too.

Reply:
Goo point. Which other types are you thinking of? To me, the only other types that seem reasonable to use as the basis of a combobox are "email" and "url". Beyond that, "tel" _might_ be worth supporting.

Reply:
I added "email", "url", and "tel" to the excepted input types.